requirements/runtime.txt

```diff
@@ -14,6 +14,5 @@ tqdm
 onnx>=1.7.0
 onnxoptimizer
 onnxruntime>=1.1.2
-lxml
```

Comment:
is it worth adding `defusedxml>=0.7.1` here?

Reply:
The defusedxml=0.7.1 was installed as a dependency for other libraries. Do you suggest double-checking it and adding a certain version to the requirements anyway?

Reply:
no